Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM python:3.6
 
 ENV PYTHONUNBUFFERED 1
 RUN mkdir /code
+RUN apt-get update -y && apt-get install -y wget \
```

Comment:
Out of interest, what are we using wget for?

Reply:
That was me copying from the IIIF pipeline's Docker image and not fully realizing that I could remove wget^